### PR TITLE
[Topic Entry] Sklearn:Support-Vector-Machines

### LINKED ISSUE #4860: [Concept Entry] Sklearn support-vector-machines
### Reason/inspiration (optional)

We would like a new entry on the `Support Vector Machines` concept under Sklearn. The entry should go in a new file under `docs/content/sklearn/concepts/support-vector-machines/support-vector-machines.md`.

The entry should include:

- An introduction to the concept
- A `Syntax` section that provides the syntax for the concept (if applicable)
- An `Example` section that provides an example demonstrating the concept in use
- A `Codebyte Example` section that provides a codebyte example demonstrating the concept in use

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by liamjdavis**
### Description

Added a new topic for Support Vector Machines with Sklearn, as well as adding 'supervised learning' and 'unsupervised learning' to tags.

### Issue Solved

Closes #4860 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ ] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

File: content/sklearn/concepts/support-vector-machines/support-vector-machines.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+- A lower C value prioritizes reducing training error, but might result in a smaller margin and poorer generalization on unseen data.
+
+Choosing the optimal C value often involves experimentation and techniques like grid search or cross-validation. In terms of choosing the best combination of kernel and C value, here are a few general guidelines:
+
+- For linearly separable data, use the 'linear' kernel.
+- For moderately complex, non-linear data, consider 'rbf' or a low-degree 'poly' kernel with a moderate C value.
+- For highly complex data, experiment with different kernels and C values using techniques like grid search.
+
+## Codebyte Example
+
```
Comment (avdhoottt) [inline content/sklearn/concepts/support-vector-machines/support-vector-machines.md:55]: Write a line here describing the code.

File: content/sklearn/concepts/support-vector-machines/support-vector-machines.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+The C parameter in SVC controls the trade-off between maximizing the margin and reducing training error. 
+
+- A higher C value prioritizes a larger margin, potentially leading to overfitting on the training data. 
+- A lower C value prioritizes reducing training error, but might result in a smaller margin and poorer generalization on unseen data.
+
+Choosing the optimal C value often involves experimentation and techniques like grid search or cross-validation. In terms of choosing the best combination of kernel and C value, here are a few general guidelines:
+
+- For linearly separable data, use the 'linear' kernel.
+- For moderately complex, non-linear data, consider 'rbf' or a low-degree 'poly' kernel with a moderate C value.
+- For highly complex data, experiment with different kernels and C values using techniques like grid search.
```
Comment (avdhoottt) [inline content/sklearn/concepts/support-vector-machines/support-vector-machines.md:52]:
```suggestion
- For linearly separable data, use the `linear` kernel.
- For moderately complex, non-linear data, consider `rbf` or a low-degree `poly` kernel with a moderate C value.
- For highly complex data, experiment with different kernels and C values using techniques like grid search.
```

File: content/sklearn/concepts/support-vector-machines/support-vector-machines.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  - 'Supervised Learning'
+  - 'Sci-kit Learn'
+CatalogContent:
+  - 'getting-started-with-python-for-data-science'
+  - 'paths/data-science'
+---
+
+**Support Vector Machines**(SVMs) are a supervised learning algorithm excelling at classification tasks. They work by finding the optimal hyperplane that maximizes the margin between different classes in your data. The margin is the distance between the hyperplane and the closest data points from each class, called support vectors. SVMs are particularly effective for high-dimensional datasets, and handle complex data while promoting good generalization and reducing the risk of overfitting.
+
+## Example
```
Comment (avdhoottt) [inline content/sklearn/concepts/support-vector-machines/support-vector-machines.md:17]: Move the example section after the `Syntax` subheading. The example section should contain a code block and its output.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Hey @liamjdavis, Excellent entry! I've suggested some changes, please make them. Thanks!!

**Comment by liamjdavis** [comment]
@avdhoottt I've edited the entry with your suggestions. Please take another look as soon as you can. Thank you.

**Review by avdhoottt — APPROVED** [review]
LGTM!!

**Review by Sriparno08 — APPROVED** [review]
Looks good, @liamjdavis! 👍

**Pull request merged**